android/app/src/main/res/layout/fragment_editor.xml

```diff
@@ -284,15 +283,17 @@
         android:orientation="vertical"
         android:padding="@dimen/margin_base">
         <TextView
-          android:layout_width="match_parent"
-          android:layout_height="32dp"
-          android:text="Aditionally open a discussion note (optional)"
-          android:textAppearance="@style/MwmTextAppearance.Body2"/>
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:fontFamily="@string/robotoMedium"
+          android:text="Open a discussion note"
+          android:textAppearance="@style/MwmTextAppearance.Body3"
+          tools:ignore="UnusedAttribute"/>
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginTop="@dimen/margin_half"
-            android:text="Your note will be published on OSM. Please describe errors or additional details that cannot be edited in Organic Maps, so other OSM volunteers can help to solve the problem."
+            android:text="Use this to start a discussion with other OSM volunteers. Please describe errors or additional details that cannot be edited in Organic Maps."
```

Comment:
Is this text replaced by another from strings.txt?

Reply:
I now added `editor_note_description` to strings.txt